lib/t/push_notifications/apns.ex

```diff
@@ -87,11 +87,18 @@ defmodule T.PushNotifications.APNS do
   end
 
   def build_alert_payload("match_about_to_expire" = type, data) do
-    %{"name" => name} = data
+    %{"name" => name, "gender" => gender} = data
+
+    body =
+      case gender do
+        "F" -> dgettext("apns", "Contact her ✨")
+        "M" -> dgettext("apns", "Contact him ✨")
+        "N" -> dgettext("apns", "Contact them ✨")
```

Comment:
I prefer templating pronouns / verbs / etc.

```elixir
body = dgettext("Contact %{pronoun} ✨", pronoun: pronoun(gender))
```
```gettext
defp pronoun("F"), do: dgettext("apns", "her")
defp pronoun("M"), do: dgettext("apns", "him")
defp pronoun("N"), do: dgettext("apns", "them")
```

This approach can be used for all messages.

Reply:
gettext doesn't seem to like this approach (I had similar problem when fixing "news" localization):

<img width="449" alt="image" src="https://user-images.githubusercontent.com/16062322/157775168-d6104fb0-73c9-4f6a-bcd3-79e138b2df72.png">

Reply:
I missed the domain in my example.

```elixir
body = dgettext("apns", "Contact %{pronoun} ✨", pronoun: pronoun(gender))
```